garden_ai/constants.py

```diff
@@ -1,3 +1,13 @@
+import os
+
+_garden_dir = os.path.expanduser("~/.garden")
+
+
 class GardenConstants:
```

Comment:
I think an Enum is clumsier because you would have to say GardenConstants.Garden_DIR.value. But I'd be open to just exposing the values directly. So in other files usage would look like ...

```
from garden_ai import garden_constants

print(garden_constants.GARDEN_DIR)
```